src/components/Main.js

```diff
@@ -6,7 +6,7 @@ import Dashboard from './Dashboard';
 import Footer from './Footer';
 import Landing from './Landing';
 import Messaging from './Messaging';
-import Nav from './Nav';
+import Header from './Header';
```

Comment:
If we're no longer using the Nav component, will we be removing it? Or is there a way we can still use and implement Nav in this component?

For example:
```
<header>
    // ...
    <Nav />
    // ...
</header>
```

This way we can group this. The navigation items are technically still a nav bar per se, We could keep it in the Nav component, and just update the Nav component to reflect the styling presented in the mockup. The header could also contain other things like Organization Logo, or jumbotron image/background image, etc.

Reply:
We have to redo the Nav then, cause if we're removing material UI we're getting rid of pretty much everything in there. the links in the nav are within MenuItem which is part of material UI

Reply:
Yep, let's just add removing MUI on this ticket then. 

Structure of the Nav is gonna be pretty much what you have in your Header so there's not going to be much rework.